calico_cni/calico_cni.py

```diff
@@ -336,14 +357,18 @@ def _create_endpoint(self, assigned_ip):
                                                     self.container_id,
                                                     [assigned_ip])
         except AddrFormatError:
```

Comment:
We can probably combine these two branches, since they are performing the same thing.

For example:
```
try:
  ...
except (AddrFormatError, KeyError), e:
  ... do cleanup ...
  self._print_error_response(ERR_CODE_GENERIC, e.message)
```

We can split again if we want to use different error codes, but I don't think that we really care about differentiating these cases on the CNI - they're both still "failed to create endpoint".